app/styles/partials/event-card.scss

```diff
@@ -36,3 +36,8 @@
     height: 130px;
   }
 }
+
+.display-block {
+  display: block;
+}
+
```

Comment:
@Anupam-dagar If you just fix the .event.card to 365 px in this file, Your work will be done and all this much wont be needed .

Reply:
@kushthedude I know that. But doing that will be just assuming the max height of the card. Also I did a code refactor, this involves less conditional decision making via `if` and explicit definition for mobile version. You should take a look here once https://semantic-ui.com/views/card.html.
Just for the pointer, following will happen if I put the height to 365px, it must be atleast 380px.
![Screenshot from 2019-03-27 01-40-49](https://user-images.githubusercontent.com/21174572/55030026-751d8800-5031-11e9-81f8-de62ec15a726.png)

Reply:
I think then you need to add a test file for the file you added .

Reply:
@kushthedude I have fixed it :)

Reply:
Great lets wait for Travis Review 😄 👍